src/app/components/Modals/PendingVerificationModal.tsx

```diff
@@ -0,0 +1,71 @@
+import { FaCheckCircle } from "react-icons/fa";
+
+import { getNetworkConfig } from "@/config/network.config";
+
+import { GeneralModal } from "./GeneralModal";
+
+interface PendingVerificationModalProps {
+  open: boolean;
+  onClose: (value: boolean) => void;
+  verified: boolean;
+  onStake?: () => void;
+  awaitingWalletResponse: boolean;
+}
+
+const Verified = () => (
+  <>
+    <FaCheckCircle className="text-5xl text-success" />
+    <h3 className="text-xl text-center">Verified</h3>
+    <p className="text-sm text-center">
+      Your request has been verified by the babylon blockchain. You can now
+      stake
+    </p>
+  </>
+);
+
+const NotVerified = () => (
+  <>
+    <span className="loading loading-spinner loading-lg text-primary" />
+    <h3 className="text-xl text-center">Pending Verification</h3>
+    <p className="text-sm text-center">
+      The babylon blockchain has received your request. Please wait while we
+      confirm the neseccary amount of signatures
+    </p>
+  </>
+);
+
+export function PendingVerificationModal({
+  open,
+  onClose,
+  verified,
+  onStake,
+  awaitingWalletResponse,
```

Comment:
Naming: awaitingWalletResponse -> loading

Reply:
@totraev but we use `awaitingWalletResponse` in such cases for the staking and unbonding/withdrawal
So `loading` in this particular modal is just the flag. While not `verified` it will be in loading state
But `awaitingWalletResponse` is when we click the stake button and waiting for ...well... wallet response :)

So if we go with `loading` then we should rename this particular variable in different places also

Reply:
> So if we go with loading then we should rename this particular variable in different places also

@gbarkhatov this is not true. Why our modal should know anything about `WalletResponse`? You can use loading state along side with other var names:

```
<Modal loading={awaitingWalletResponse} />
```